features/admin.branding.v1/components/branding-page-layout.tsx

```diff
@@ -263,14 +264,28 @@ const BrandingPageLayout: FunctionComponent<BrandingPageLayoutInterface> = (
                                                 value={ brandingMode }
                                                 disabled={ isBrandingAppsRedirect }
                                             >
-                                                <ToggleButton value={ BrandingModes.ORGANIZATION }>
+                                                <ToggleButton
+                                                    value={ BrandingModes.ORGANIZATION }
+                                                    onClick={ () => {
+                                                        updateActiveTab(
+                                                            BrandingPreferencesConstants.TABS.GENERAL_TAB_ID
```

Comment:
Do we need to navigate to general tab even when the user is in org branding?

Reply:
To maintain the consistency in the behaviour..